pkg/fileutil/fileutil.go

```diff
@@ -121,7 +123,18 @@ func CopyTree(src, dest string) error {
 			return fmt.Errorf("unsupported mode: %v", mode)
 		}
 
-		if err := os.Lchown(target, int(info.Sys().(*syscall.Stat_t).Uid), int(info.Sys().(*syscall.Stat_t).Gid)); err != nil {
+		var srcUid uint64 = uint64(info.Sys().(*syscall.Stat_t).Uid)
+		var srcGid uint64 = uint64(info.Sys().(*syscall.Stat_t).Gid)
+
+		if uidrange.UidCount > 0 && (srcUid >= uidrange.UidCount || srcGid >= uidrange.UidCount) {
```

Comment:
How about putting this check and the one below to member function of `UidRange`?